vite.config.js

```diff
@@ -80,9 +88,6 @@ export default defineConfig(({ mode }) => {
   return {
     plugins: [
       inlineHbsCompile(),
-      babel({
-        exclude: '**/formio.form.min*',
-      }),
```

Comment:
We don't need this exclude anywhere anymore? For `formio.form.min`?

Reply:
nope.  the vite plugin ignores node_modules by default